docs/conf.py

```diff
@@ -126,26 +124,36 @@
             },
         },
     ],
-}
+    "version_dropdown": True,
+    "version_info": [
+        {
+            "version": "https://sphinx-immaterial.rtfd.io",
+            "title": "ReadTheDocs",
+            "aliases": ["RTD"]
```

Comment:
Note: `aliases` are for specifying alternative directory names to match against the URL in order to figure out what the current version is.  They aren't for displaying to the user.

For example, if you have:

```
1.0/
2.0/
3b1/
```

and your webserver additionally redirects `stable/` -> `2.0/` and `latest/` to `3b1/`  then you should make `stable` and alias of `2.0` and `latest` an alias of `3b1`.

I'm guessing in this case you don't intend for `RTD` to be the actual path.

Reply:
I want really sure what aliases we're for (until now).

Reply:
I used this info in hopes of resolving the comment below about explaining the use of the `aliases` field.